public/docs/ts/latest/guide/structural-directives.jade

```diff
@@ -0,0 +1,647 @@
+include ../../../../_includes/_util-fns
+
+:marked
+  One of the defining features of a single page application is its manipulation
+  of the DOM tree. Instead of serving a whole new page every time a user
+  navigates, whole sections of the DOM disappear and are reconstructed according 
+  to the application state. In this chapter we'll to look at how Angular
+  does this DOM manipulation, and how we can do it ourselves in our own directives.
+  
+  ### Table of contents
+  
+  - [Structural directives and element instantiation](#structural-directives-and-element-instantiation)
+  - [Presence versus visibility](#presence-versus-visibility)
+  - [Behind the hood: templates](#behind-the-hood-templates)
+  - [Making our own structural directive](#making-our-own-structural-directive)
+  - [The code so far](#the-code-so-far)
+  - [Configurable component DOM](#configurable-component-dom)
+  - [Heroes Dashboard](#heroes-dashboard)
+  - [A tale of three DOMs](#a-tale-of-three-doms)
+  - [Element projection](#element-projection)
+  - [Templates for components](#templates-for-components)
+  - [Components as first-class citizens](#components-as-first-class-citizens)
+  - [Finished dashboard](#finished-dashboard)
+  - [Wrapping up](#wrapping-up)
+  
+  A live example of all the code in this chapter can be found in [this plunker](/resources/live-examples/structural-directives/ts/src/plnkr.html).
```

Comment:
The standard is simply [Live example](/resources/live-examples/structural-directives/ts/src/plnkr.html)